themes/default/content/blog/2022-03-30-introducing-pulumiverse/index.md

```diff
@@ -0,0 +1,77 @@
+
+This project is especially important to me, because it's an idea that I've tried in the past, 2018 😯, and failed to commit to and give it the attention it deserves.
+
+Fortunately, two other wonderful Pulumi community members stepped up and started to put together their own programme and with me now being at Pulumi, the timing was amazingly serendipitous.
+
+I spoke with [Ringo De Smet](https://twitter.com/ringods), Head of CI/CD at Napoleon Games, and founding Pulumiverse member and he had this to say about why he's excited to be founding the Pulumiverse.
+
+> For infrastructure automation, we had other great tools which preceded Pulumi: Chef, Puppet, Ansible. Any of these tools have a vibrant community of people preventing that all of us need to re-invent the wheel. We had Sous-Chefs[1] for Chef & Vox Pupuli[2] for Puppet. But I see a lot of Pulumi users implementing the same stuff over and over again. That's why I think Pulumi needs it's own community. So I started it: Pulumiverse[3]
+
+[1] https://sous-chefs.org
+[2] https://voxpupuli.org
+[3] https://github.com/pulumiverse
+
+## What is the Pulumiverse?
+
+Pulumiverse, in the simplest form, is a GitHub organization. It provides a community operated place for anyone to communicate, contribute, and collaborate with other members of the Pulumi community and build abstractions and SDKs that can be shared and used by anyone.
+
+There are many libraries or collections of components for Pulumi out there. Most of them are made based on requirements of individual companies and developers. Pulumiverse aims to make a community where these libraries can be created, where opinionated configurations for Pulumi resources can be stored and discussed.
+
+## Current Projects
+
+The Pulumiverse launches with just a few projects, but we know that with the support of such an amazing community, the Pulumiverse will grow and grow and provide invaluable resources for anyone who wants to build or use Pulumi for a library or project.
+
+### Awesome Pulumi
+
+[Awesome Pulumi](https://github.com/pulumiverse/awesome-pulumi)
+
+Everybody loves an awesome list and this one delivers you a list of awesome Pulumi resources, books, tutorials, talks, and much more.
+
+Want your project listed? Open a PR. We can't wait to share it with everyone.
+
+### Kubernetes SDKs
+
+[Kubernetes SDKs](https://github.com/pulumiverse/kubernetes-sdks)
+
+The Kubernetes SDKs project aims to provide multi-language SDKs for interacting with Kubernetes Custom Resource Definitions (CRDs). Currently, it automatically builds SDKs for:
+
+- ArgoCD
+- CertManager
+- Contour
+- Crossplane
+- Istio
+- Kong Ingress
+- Knative
+- RedPanda
+- Tyk
+
+This is just the beginning and we want to expand this collection quickly. So if you're working with Kubernetes and want a rich developer expeirence for authoring custom resources, open an issue and let us know what you need.
+
+### Pulumi Unifi
+
+[Unifi Provider](https://github.com/pulumiverse/pulumi-unifi)
```

Comment:
For these last two, any blurb you can add? They seem a bit tacked on at the end without some introduction (even if it's something like "and here's a community provider example blah blah blah."

Reply:
I wanted to ... but I don't really know what they do 😅

Reply:
Then how about something like this:

```suggestion
[Unifi Provider](https://github.com/pulumiverse/pulumi-unifi)

This community-maintained provider is one example of how you can share your providers with the world as part of the Pulumiverse.
```